tasks/publish.yaml

```diff
@@ -0,0 +1,50 @@
+variables:
+  - name: TARGET_REPO
+    default: oci://ghcr.io/defenseunicorns/packages/uds
+
+  - name: VERSION
+    description: "The version of the packages to build"
+    # x-release-please-start-version
+    default: "0.1.0"
+    # x-release-please-end
+
+tasks:
+  - name: packages
```

Comment:
I would prefer if we use the same terminology for inputs to UDS Bundles. Mattermost and UDS Core are Zarf packages that we bundle. Different terms for them adds some confusion

Reply:
Agree we need to sort this, and as discussed, will address in another PR.